src/stcal/skymatch/skystatistics.py

```diff
@@ -0,0 +1,136 @@
+"""
+The `skystatistics` module provides statistics computation class used by
+:py:func:`~stcal.skymatch.skymatch.match`
+and :py:class:`~stcal.skymatch.skyimage.SkyImage`.
+"""
+
+from copy import deepcopy
+
+# THIRD PARTY
+from stsci.imagestats import ImageStats  # type: ignore[import-untyped]
```

Comment:
might be worth considering making the relevant third-party type ignores global by putting them here https://github.com/spacetelescope/stcal/blob/d6a87c346b56daafcd6b8b43231fdbb8b9222965/pyproject.toml#L212